packages/core/src/components/forms/radioGroup.tsx

```diff
@@ -20,6 +20,11 @@ export interface IRadioGroupProps extends IProps {
      */
     disabled?: boolean;
 
+    /**
+     * True if radio buttons are to be displayed inline horizontally.
```

Comment:
please match language for other boolean props, like the one above which begins "Whether..." (like you did in the file above)